exporter/clickhouseexporter/exporter_metrics_test.go

```diff
@@ -148,6 +148,7 @@ func simpleMetrics(count int) pmetric.Metrics {
 	sm.Scope().SetDroppedAttributesCount(10)
 	sm.Scope().SetName("Scope name 1")
 	sm.Scope().SetVersion("Scope version 1")
+	timestamp := time.Unix(1703498029, 0)
```

Comment:
How is this related to the integration test?

Reply:
integration test will verify the inserted data, so we need a constant timestamp